typewritten.zsh

```diff
@@ -65,44 +65,49 @@ tw_get_home_relative_wd() {
   echo "%F{$tw_current_directory_color}%~"
 }
 
-tw_get_git_relative_wd() {
+tw_get_displayed_wd() {
   local tw_home_relative_wd="$1"
   local tw_git_branch="$2"
   local tw_git_home="$3"
 
-  local tw_git_relative_wd="%F{$tw_current_directory_color}$tw_git_home%c"
+  local tw_displayed_wd="%F{$tw_current_directory_color}$tw_git_home%c"
+
+# The pure layout defaults to home relative working directory, but allows customization
+  if [[  "$TYPEWRITTEN_PROMPT_LAYOUT" = "pure" && "$TYPEWRITTEN_RELATIVE_PATH" = "" ]]; then
+    tw_displayed_wd=$tw_home_relative_wd
+  fi;
 
   if [[ "$TYPEWRITTEN_RELATIVE_PATH" = "home" ]]; then
-    tw_git_relative_wd=$tw_home_relative_wd
+    tw_displayed_wd=$tw_home_relative_wd
   fi;
 
   if [[ "$TYPEWRITTEN_RELATIVE_PATH" = "adaptive" ]]; then
     if [[ "$tw_git_branch" = "" ]]; then
-      tw_git_relative_wd=$tw_home_relative_wd
+      tw_displayed_wd=$tw_home_relative_wd
     fi;
   fi;
 
-  echo $tw_git_relative_wd
+  echo $tw_displayed_wd
 }
 
 tw_redraw() {
   tw_home_relative_wd="$(tw_get_home_relative_wd)"
-  tw_git_relative_wd="$(tw_get_git_relative_wd $tw_home_relative_wd $tw_prompt_data[tw_git_branch] $tw_prompt_data[tw_git_home])"
+  tw_displayed_wd="$(tw_get_displayed_wd $tw_home_relative_wd $tw_prompt_data[tw_git_branch] $tw_prompt_data[tw_git_home])"
```

Comment:
suggestion(non-blocking): Since `tw_home_relative_wd` is now only used inside of the `tw_get_displayed_wd` function, let's move the variable inside and not pass it as a param anymore